### ISSUE TITLE
[TW-1111] duplicate command should display the ID of the created task

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1146]
_Cory Donnelly on 2009-12-01T11:00:59Z says:_ 

 Returning the id of the source task is nice, but it would be more helpful if the newly-created task was also displayed:

<pre>
$ task duplicate 62
Duplicated 62 'foo'
Duplicated 1 task
</pre>

It should be consistent with the add command:

<pre>
$ task add foo
Created task 85
</pre>

Proposed:

<pre>
$ task duplicate 62
Duplicated 62 'foo'
Duplicated 1 task
Created task 85
</pre>

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2009-12-01T11:00:59Z
Modified: 2014-02-09T02:31:52Z
```

**Comment by taskwarrior** [comment]
_David Patrick on 2009-12-01T11:22:45Z says:_ 

 agreed

**Comment by taskwarrior** [comment]
_Federico Hernandez on 2009-12-01T16:44:35Z says:_ 

 Fixed for upcoming 1.8.5

/Fredde

<pre>
[ultrafredde(at)cottonwood tmp]$ ./task-185 rc:x ls
Using alternate .taskrc file x

ID Project Pri Description
 1             AAA
 2             BBB
 3             CCC
 4             DDD

4 tasks
[ultrafredde(at)cottonwood tmp]$ ./task-185 rc:x duplicate 2
Using alternate .taskrc file x
Duplicated 2 'BBB'
Duplicated 1 task
Created task 5
[ultrafredde(at)cottonwood tmp]$ ./task-185 rc:x ls
Using alternate .taskrc file x

ID Project Pri Description
 1             AAA
 2             BBB
 3             CCC
 4             DDD
 5             BBB

5 tasks
</pre>
